testsuite/MDAnalysisTests/test_persistence.py

```diff
@@ -36,10 +36,8 @@
 import shutil
 import warnings
 
-"""These tests need to be reworked for the new 363 topoloy system
 
-"""
-class SkipAtomGroupPickle(object):
+class TestAtomGroupPickle(object):
```

Comment:
So some of these are failing, but I wasn't sure that they should actually pass...

There's a Universe created like this:
```python
        self.universe_n = MDAnalysis.Universe(PDB_small, PDB_small, PDB_small,
                                              is_anchor=False,
                                              anchor_name="test1")
```

Which looks like it's declared as not being an anchor, but later is expected to be unpickled against.  It seems the `anchor_name` seems to override the `is_anchor` keyword?